AsyncDisplayKit/ASPagerNode.h

```diff
@@ -82,14 +86,15 @@
 /**
  * Data Source is required, and uses a different protocol from ASCollectionNode.
  */
-- (void)setDataSource:(id <ASPagerDataSource>)dataSource;
-- (id <ASPagerDataSource>)dataSource;
+- (void)setDataSource:(nullable id <ASPagerDataSource>)dataSource;
+- (nullable id <ASPagerDataSource>)dataSource;
 
 /**
  * Delegate is optional, and uses the same protocol as ASCollectionNode.
  * This includes UIScrollViewDelegate as well as most methods from UICollectionViewDelegate, like willDisplay...
  */
-@property (nonatomic, weak) id <ASPagerDelegate> delegate;
```

Comment:
Let's update this comment since `ASPagerDelegate` is no longer the same protocol that ASCollectionNode uses.